docs/docs/fallback-handoff.mdx

```diff
@@ -185,21 +157,33 @@ policies:
 ```
 
 :::note
-
 If you do not want the `Rule Policy` to predict a default action in case of low Core
 confidence, specify `enable_fallback_prediction: False` in the configuration of the
 policy.
 :::
 
 
-`action_default_fallback` is a default action in Rasa Open Source that sends the
-`utter_default` response to the user. Make sure to specify
+#### 2. Defining the default response message
+
+By default, Rasa Open Source will send the `utter_default` response to the user when
+Core confidence falls below the configured threshold. Make sure to specify
 the `utter_default` in your domain file. It will also revert back to the
 state of the conversation before the user message that caused the
 fallback, so it will not influence the prediction of future actions.
 
-You can also create your own custom action to use as a fallback (see
-[Custom Actions](./actions.mdx#custom-actions) for more info on custom actions).
+`utter_default` can be defined in your domain file as:
+
+```yaml
+responses:
+  utter_default:
+  - text: Sorry I didn't get that. Can you rephrase?
+```
+
+#### 3. Customizing the default action (optional)
+
+`action_default_fallback` is a default action in Rasa Open Source that sends the
```

Comment:
i don't think Rasa Open Source needs to be specified here, you can just say it's used by default